frodo.js

```diff
@@ -1,56 +1,54 @@
-const truncatedString = string.substring(0, maxLength) + (string.length > maxLength ? "..." : "");
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.